core/external/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "near-external"
```

Comment:
I'd call this something along the lines of `near-external-storage`

simply `near-external` feels.. too generic?